internal/agent/install.go

```diff
@@ -89,20 +87,24 @@ func ManualInstall(c string, options map[string]string, strictValidations, debug
 	mergeOption(configStr, options)
 
 	if options["device"] == "" {
-		if cc.Install.Device == "" {
-			options["device"] = detectDevice()
-		} else {
-			options["device"] = cc.Install.Device
+		// IF cc is nil we will panic, check beforehand.
+		// If we set no device then we will exit later down the line anyway
+		if cc.Install != nil {
+			if cc.Install.Device == "" {
+				options["device"] = detectDevice()
+			} else {
+				options["device"] = cc.Install.Device
+			}
 		}
 	}
 
+	// Set debug from here already, so it's loaded by the ReadConfigRun
+	viper.Set("debug", debug)
```

Comment:
If we are ok with sharing the value of `debug` using viper (like we do for `ReadConfigRun`), why don't we set this at the top level (in `main.go`, manual-install action) so we don't need to pass it as an argument to `ManualInstall`?